dev/Expander/Expander.xaml

```diff
@@ -142,12 +141,12 @@
                             Content="{TemplateBinding Header}"
                             ContentTemplate="{TemplateBinding HeaderTemplate}"
                             ContentTemplateSelector="{TemplateBinding HeaderTemplateSelector}"
-                            HorizontalAlignment="{TemplateBinding HorizontalAlignment}"
+                            HorizontalAlignment="Stretch"
```

Comment:
HorizontalContentAlignment

Reply:
True that both Horizontal and VerticalContentAlignment of the ExpanderHeader need to be stretched. That's one of those areas I didn't address. 

I will go over the entire template again as late last night it was just quick changes to address my issue directly. We shouldn't be setting Min/MaxHeight/Width everywhere too. The control itself (the Grid) should have those constraints and all intermediate elements including the ExpanderHeader and ExpanderContent should automatically size. The remaining properties go directly to the ContentPresenter.

Reply:
I believe this is correct. See the comments for the ExpanderContent. Both of these intermediate controls should size to the layout root grid.

Reply:
Ok agreed. That makes sense now.

Reply:
I also stretched the HorizontalContentAlignment of the header and explicitly set VerticalContentAlignment to Center (since it's required to align with the chevron). This should actually close #4248 now @michael-hawker correct me if I'm wrong.